lib/generators/open_stax/rescue_from/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails/generators'
+
+module OpenStax
+  module RescueFrom
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def copy_templates
+        copy_file "any.html.erb", "app/views/errors/any.html.erb"
+      end
+
+      def copy_initializer
+        copy_file "rescue_from.rb", "config/initializers/rescue_from.rb"
+      end
+
+      def configure_environments
+        test_file = "test.rb"
+        test_middleware = <<-TEST
+  config.middleware.use ExceptionNotification::Rack, email: {
+    email_prefix: "[Tutor] (TEST) ",
+    sender_address: %{"OpenStax Tutor" <noreply@openstax.org>},
```

Comment:
in general, even tho this is an openstax gem, we don't really want a bunch of OpenStax-specific things in here (no real reason for it to be when it can all be in the config)